presto-main/src/test/java/io/prestosql/sql/planner/iterative/rule/TestPruneAggregationColumns.java

```diff
@@ -74,7 +74,7 @@ private ProjectNode buildProjectedAggregation(PlanBuilder planBuilder, Predicate
                 planBuilder.aggregation(aggregationBuilder -> aggregationBuilder
                         .source(planBuilder.values(key))
                         .singleGroupingSet(key)
-                        .addAggregation(a, planBuilder.expression("count()"), ImmutableList.of())
-                        .addAggregation(b, planBuilder.expression("count()"), ImmutableList.of())));
+                        .addAggregation(a, PlanBuilder.expression("count()"), ImmutableList.of())
+                        .addAggregation(b, PlanBuilder.expression("count()"), ImmutableList.of())));
```

Comment:
do we typically static-import these?

(the `Access PlanBuilder.expressions in static manner` commit adds static import in some cases and qualified call in some other places, intentional?)

Reply:
I added static import when possible. In some cases it conflicted with `ExpressionMatcher expression`. Maybe this is the reason to not static import at all?

Reply:
Good question. this is "whatever" for now for me, we can talk about this sepearete from this PR